mobzy-nms-injection/src/main/kotlin/com/mineinabyss/mobzy/injection/types/FishMob.kt

```diff
@@ -24,6 +29,24 @@ open class FishMob(
     //on player interact
     override fun b(entityhuman: NMSEntityHuman, enumhand: NMSHand) = NMSInteractionResult.d //PASS
 
+    init {
+        a(PathType.i, 0.0F)
+        bM = SmoothSwimmingMoveControl(this, 85, 10, 1.0f, 0.5f, false)
+        bL = SmoothSwimmingLookControl(this, 20)
+    }
+
+    override fun g(movementInput: Vec3D) {
+        if (doAITick() && this.isInWater) {
+            this.a(0.1f, movementInput)
```

Comment:
Think stuff exists for ControllerMove but not for the Vector3D stuff used there ithink?
Not entirely sure what it would be u would wanna replace there either